src/rebar_packages.erl

```diff
@@ -22,57 +25,93 @@
 -type vsn() :: binary().
 -type package() :: pkg_name() | {pkg_name(), vsn()}.
 
--spec packages(rebar_state:t()) -> ets:tid().
-packages(State) ->
-    catch ets:delete(?PACKAGE_TABLE),
```

Comment:
Of the delete? There is a `packages/2` if you just mean the returning of the `ets:tid()` for a registry.